Src/MoneyFox.Ui/Views/Settings/SettingsViewModel.cs

```diff
@@ -31,4 +41,33 @@ public CurrencyViewModel SelectedCurrency
     }
 
     public IReadOnlyList<CurrencyViewModel> AvailableCurrencies { get; }
+
+    public string SelectedAccount
+    {
+        get => selectedAccount;
+
+        set
+        {
+            SetProperty(field: ref selectedAccount, newValue: value);
+            settingsFacade.DefaultAccount = selectedAccount;
+            OnPropertyChanged();
+        }
+    }
+
+    public void LoadAccounts()
+    {
+        availableAccounts = mediator.Send(new GetAccountsQuery()).Result.Select(a => a.Name).ToList();
+    }
+
+    public IReadOnlyList<string>? AvailableAccounts
```

Comment:
I think this should not be nullable. If there are not data it would just be an empty list, right?

Reply:
I just found that picker only accepts `IList` for bindings. So if the IReadOnlyList doesn't work and you have to go back to a List sorry for steering you wrong.

Reply:
Made it a list and non-nullable.